client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -426,6 +431,17 @@ export class List extends React.Component {
 				/>
 			);
 		} );
+
+		if ( hasSingleSite || renderAllSites || ! config.isEnabled( 'manage/all-domains' ) ) {
+			return domainListItems;
+		}
+
+		return [
+			...domainListItems,
+			<CompactCard href={ domainManagementRoot() }>
```

Comment:
@eltongo, looks like this addition is causing a warning to be thrown in dev mode:

```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `List`. See https://fb.me/react-warning-keys for more information.
    in CompactCard (created by List)
    in List (created by Context.Consumer)
    in WithLocalizedMoment(List) (created by Localized(WithLocalizedMoment(List)))
    in Localized(WithLocalizedMoment(List)) (created by ConnectFunction)
    in ConnectFunction (created by StoreConnection)
    in StoreConnection (created by DomainManagementData)
    in div (created by DomainManagementData)
```

That's because this `CompactCard` does not have a `key`. Something like `key='manage-all-domains'` should be sufficient - since that component is the same regardless of selected site.
```

Reply:
Fixed in #43456

Thanks 🙇

Reply:
Thank you!